### PR TITLE
[IMP] hr_recruitment: UX improvements

### PR DESCRIPTION
**Pull request by sofiagvaladze**
Add an Interview parts on applications, Multi-Application Management and UX improvements.

Improve in Recruitment app Job Positions Kanban view;
It is more intuitive if list is applications default view; improve the applications form;

In applications form for the Other applicants stat button: the other applicants for the same
applicant are found using only e-mail. Change it and find other applications using email,
name and number of an applicant.
Add interview template on recruitment settings and applications form view.

Task - 2404610

### CODE REVIEW

**Review by kbapt — CHANGES_REQUESTED** [review]

File: addons/web/static/src/legacy/js/fields/basic_fields.js [hunk 1]
```diff
@@ -1741,6 +1741,7 @@ var UrlWidget = InputField.extend({
             href: href,
             target: '_blank',
         });
+        this.el.textContent = '';
```
Comment (SimonGenin) [inline addons/web/static/src/legacy/js/fields/basic_fields.js:1744]: Why do you need this ? It's "dangerous" to touch these files :)

**Review by SimonGenin — CHANGES_REQUESTED** [review]

File: addons/web/static/src/legacy/js/fields/basic_fields.js (hunk 1 above)
Comment (sofiagvaladze) [reply addons/web/static/src/legacy/js/fields/basic_fields.js:1744]: Hey @SimonGenin, (sorry for late reply)

So, in Recruitment app, when we create an tracker, let's say we first choose Facebook, then Twitter... The URL parameter of Twitter does not replace Facebook URL, instead it is added to it. All of this before we save it. So we see long unmeaningful URL. That's why this. What do you think? Can we do it some other way?

Comment (aab-odoo) [reply addons/web/static/src/legacy/js/fields/basic_fields.js:1849]: Fix seems ok, but you should write a test

**Review by SimonGenin — APPROVED** [review]

**Comment by kba-odoo** [comment]
@robodoo r+

**Pull request closed without merging**